src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java

```diff
@@ -407,13 +410,13 @@ void testGettingHearingVenueAddressesHtml() {
 
     @Test
     void testGeneratingHearingVenueList() {
-        caseDetails.getCaseData().setManagingOffice("Manchester");
+        caseDetails.getCaseData().setManagingOffice(MANCHESTER.getOfficeName());
```

Comment:
That's fine to avoid writing string in the method but once we add PMD to this project, it will complain of violation of Law of Demeter (don't talk to stranger), stranger being MANCHESTER here. It should be fine for now though

Reply:
I'll take note for future reference.